include/deal.II/base/std_cxx20/iota_view.h

```diff
@@ -0,0 +1,48 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2020 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// The deal.II library is free software; you can use it, redistribute
+// it, and/or modify it under the terms of the GNU Lesser General
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE.md at
+// the top level directory of deal.II.
+//
+// ---------------------------------------------------------------------
+#ifndef dealii_cxx20_iota_view_h
+#define dealii_cxx20_iota_view_h
+
+#include <deal.II/base/config.h>
+
+#include <boost/range/irange.hpp>
+
+
+DEAL_II_NAMESPACE_OPEN
+
+namespace std_cxx20
```

Comment:
can you please add a (short) doxygen documentation to the two namespaces so that they appear in the documentation?

Reply:
Documentation for that namespace is part of https://github.com/dealii/dealii/pull/10047.